vault/resource_mfa_duo_test.go

```diff
@@ -0,0 +1,94 @@
+package vault
+
+import (
+	"fmt"
+	"regexp"
+	"testing"
+
+	"os"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	"github.com/hashicorp/vault/api"
+)
+
+func testMFADuo_basic(t *testing.T) {
```

Comment:
```suggestion
func TestMFADuo_basic(t *testing.T) {
```

Reply:
I found that when I run this test, I get:
```
=== RUN   TestMFADuo_basic
--- FAIL: TestMFADuo_basic (0.01s)
    testing.go:538: Step 0 error: config is invalid: vault_mfa_duo.test: "push_info": required field is not set
FAIL

Process finished with exit code 1
```

Reply:
Ok, I think I've fixed it 😄